autogen_freebsd.sh

```diff
@@ -12,9 +12,10 @@ if [ x`uname`x = x"FreeBSD"x ]; then
         CXX=clang++
     fi
     CWARN=""
+    COPTIMIZE="-g -O0"
     CLANGWARN="-Wno-unused-local-typedef -Wno-mismatched-tags -Wno-macro-redefined -Wno-unused-function -Wno-unused-label -Wno-undefined-bool-conversion -Wno-unused-private-field -Wno-unused-local-typedef -Wno-uninitialized -Wno-gnu-designator -Wno-inconsistent-missing-override -Wno-deprecated-declarations -Wno-parentheses"
-    CFLAGS="-g -I/usr/local/include ${CWARN} ${CLANGWARN}"
-    CXXFLAGS="-g -DGTEST_USE_OWN_TR1_TUPLE=1 -I/usr/local/include ${CWARN} ${CLANGWARN}"
+    CFLAGS="${COPTIMZE} -I/usr/local/include ${CWARN} ${CLANGWARN}"
```

Comment:
On 22-3-2016 04:30, Kefu Chai wrote:
> In autogen_freebsd.sh
> <https://github.com/ceph/ceph/pull/7515#discussion_r56933058>:
> 
>>      CLANGWARN="-Wno-unused-local-typedef -Wno-mismatched-tags -Wno-macro-redefined -Wno-unused-function -Wno-unused-label -Wno-undefined-bool-conversion -Wno-unused-private-field -Wno-unused-local-typedef -Wno-uninitialized -Wno-gnu-designator -Wno-inconsistent-missing-override -Wno-deprecated-declarations -Wno-parentheses"
>> -    CFLAGS="-g -I/usr/local/include ${CWARN} ${CLANGWARN}"
>> -    CXXFLAGS="-g -DGTEST_USE_OWN_TR1_TUPLE=1 -I/usr/local/include ${CWARN} ${CLANGWARN}"
>> +    CFLAGS="${COPTIMZE} -I/usr/local/include ${CWARN} ${CLANGWARN}"
> 
> could you squash this change into the one you introduced autogen_freebsd.sh?

YUp, I was trying to get that to work last night, but then the whole
list with commits sort of blew up in my face. And I had to start and
rework it.
But I'll have another go at it.

--WjW